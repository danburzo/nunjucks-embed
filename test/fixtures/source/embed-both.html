{% set foo = 'bar' %}
{% embed with context "components/note.html" %}
	{% block content %}
		Hello
	{% endblock %}
{% endembed %}
{% embed without context "components/note.html" %}
	{% block content %}
		Hello
	{% endblock %}
{% endembed %}
{% set foo = 'bar2' %}
{% embed without context "components/note.html" %}
	{% block content %}
		Hello
	{% endblock %}
{% endembed %}
{% embed with context "components/note.html" %}
	{% block content %}
		Hello
	{% endblock %}
{% endembed %}