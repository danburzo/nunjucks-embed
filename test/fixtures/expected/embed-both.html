<div class='note note--default'> Hello bar </div>
<div class='note note--default'> Hello </div>
<div class='note note--default'> Hello </div>
<div class='note note--default'> Hello bar2 </div>